src/session/sessionManager.js

```
@@ -30,14 +30,13 @@ const COOKIE_LIST = [
+  try {
     const cookieStore = await cookies();
     if (!req) return appRouterSessionManager(cookieStore);
-      return isAppRouter(req)
-        ? appRouterSessionManager(cookieStore)
-        : pageRouterSessionManager(req, res);
+    return isAppRouter(req)
+      ? appRouterSessionManager(cookieStore)
+      : pageRouterSessionManager(req, res);
   } catch (error) {
-    console.error('Failed to initialize session manager:', error);
     throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)